src/core/atomic.d

```diff
@@ -24,6 +24,12 @@ else version (D_InlineAsm_X86_64)
     enum has64BitCAS = true;
     enum has128BitCAS = true;
 }
+else version (GNU)
+{
+    import gcc.config;
```

Comment:
Does GDC treat the modules `gcc.config` and `gcc.builtins` specially?

Reply:
The former is a template `config.d.in` that's generated at configure-time. The latter is an empty module that has all builtins injected into it at compile-time, its contents may vary depending on compiler flags.